x-pack/plugins/file_upload/server/routes.ts

```diff
@@ -37,7 +40,59 @@ function importData(
 /**
  * Routes for the file upload.
  */
-export function fileUploadRoutes(router: IRouter) {
+export function fileUploadRoutes(coreSetup: CoreSetup<StartDeps, unknown>) {
+  const router = coreSetup.http.createRouter();
+
+  router.get(
+    {
+      path: '/api/file_upload/has_import_permission',
```

Comment:
nit: routes not intended for public consumption should start with `/internal` instead of `/api`.